yt/frontends/ramses/hilbert.py

```diff
@@ -49,8 +50,8 @@
 
 
 def hilbert3d(
-    ijk: "np.ndarray[Any, np.dtype[np.int64]]", bit_length: int
-) -> "np.ndarray[Any, np.dtype[np.float64]]":
+    ijk: "nptype.NDArray[np.int64]", bit_length: int
+) -> "nptype.NDArray[np.float64]":
```

Comment:
This was incorrectly typed (no reason for you to know that but we might as well fix it!), `get_hilbert_indices` will return an int array, so this function will always return an int array. 

```suggestion
    ijk: "nptype.NDArray[np.int64]", bit_length: int
) -> "nptype.NDArray[np.int64]":
```